src/TableExtension.java

```diff
@@ -343,6 +383,43 @@ public Object report(Argument args[], Context context)
 
   }
 
+  public static class FromJSON implements Reporter {
+    public Syntax getSyntax() {
+      return SyntaxJ.reporterSyntax
+          (new int[]{Syntax.StringType()},
+              Syntax.WildcardType());
+    }
+
+    public String getAgentClassString() {
+      return "OTPL";
+    }
+
+    public Object report(Argument args[], Context context)
+        throws ExtensionException, LogoException {
+
+      FileManager fm = ((ExtensionContext) context).workspace().fileManager();
+
+      try {
+        String path = fm.attachPrefix(args[0].getString());
+        File file = new File(path.toString());
+        if (!file.exists()) {
+          throw new ExtensionException (args[0].get().toString() + " does not exist.");
```

Comment:
Prefer no space between a constructor and its open paren, so `ExtensionException(`